py/selenium/webdriver/firefox/options.py

```diff
@@ -87,14 +86,10 @@ def to_capabilities(self):
 
             returns a dictionary with everything
         """
-        firefox = DesiredCapabilities.FIREFOX.copy()
```

Comment:
This causes existing capabilities to be overwritten when updating them from a FirefoxOptions object. It meant that I was unable to set the `marionette` capability to `True`.